src/components/Tooltip/_tooltip.scss

```diff
@@ -36,7 +36,7 @@
 
     svg {
       fill: $text-01;
-      @include bidi-margin-left(rem(8px));
+      margin: 0 rem(8px);
```

Comment:
We'd probably want to retain the bidi functionality here. Thoughts @stpCollabr8nLstn ?

Reply:
this is bidirectional in that now the margin is on both sides (this makes the measurement where the floating menu is positioned horizontally in the middle add up correctly)

Reply:
This should be fine without the bidi mixin